code-of-conduct.html

```diff
@@ -6,23 +6,36 @@
 		<title>Code of Conduct</title>
 		<link rel="stylesheet" href="./css/common/main.css"></link>
 		<link rel="stylesheet" href="./css/pages/coc.css"></link>
-		<link rel="stylesheet" href="css/pages/navbar.css">
+		<link rel="stylesheet" href="css/pages/discord.css">
```

Comment:
I didn't get why was `navbar.css` removed and `discord.css` added to `code-of-conduct` page 🤔
Can you please explain?

If he need `header` in all files, the css should go in `main.css`
Only the page-specific css is supposed to go in the page-specific css files like `coc.css`, `discord.css`

I hope you get my point 🙂

Reply:
Yeah I am getting your point. Actually I had to remove navbar.css from all the pages cz it was creating different navbar than that of discord.css. discord.css had all the code for the navbar containing logo and login button so I added that stylesheet to every html page! If you want I can add the code from discord.css to main . This will clear the confusion !